--- a/webapp/graphite/settings.py
+++ b/webapp/graphite/settings.py
@@ -93,12 +93,16 @@
 USE_REMOTE_USER_AUTHENTICATION = False
 
 #Database settings, sqlite is intended for single-server setups
-DATABASE_ENGINE = 'sqlite3'			# 'postgresql', 'mysql', 'sqlite3' or 'ado_mssql'.
-DATABASE_NAME = STORAGE_DIR + 'graphite.db'	# Or path to database file if using sqlite3.
-DATABASE_USER = ''				# Not used with sqlite3.
-DATABASE_PASSWORD = ''				# Not used with sqlite3.
-DATABASE_HOST = ''				# Set to empty string for localhost. Not used with sqlite3.
-DATABASE_PORT = ''				# Set to empty string for default. Not used with sqlite3.
+DATABASES = {
+   'default': {
+        'ENGINE': 'django.db.backends.sqlite3', # Add 'postgresql_psycopg2', 'postgresql', 'mysql', 'sqlite3.'
+        'NAME': STORAGE_DIR + 'graphite.db',    # Or path to database file if using sqlite3.
+        'USER': '',                             # Not used with sqlite3.
+        'PASSWORD': '',                         # Not used with sqlite3.
+        'HOST': '',                             # Set to empty string for localhost. Not used with sqlite3.
+        'PORT': '',                             # Set to empty string for default. Not used with sqlite3.
+    }
+}
 
 DASHBOARD_CONF = join(CONF_DIR, 'dashboard.conf')
 GRAPHTEMPLATES_CONF = join(CONF_DIR, 'graphTemplates.conf')
