diff --git webapp/graphite/settings.py webapp/graphite/settings.py
index 0740240..a4a464d 100644
--- webapp/graphite/settings.py
+++ webapp/graphite/settings.py
@@ -157,8 +157,8 @@ SECRET_KEY = ''
 
 # List of callables that know how to import templates from various sources.
 TEMPLATE_LOADERS = (
-  'django.template.loaders.filesystem.load_template_source',
-  'django.template.loaders.app_directories.load_template_source',
+  'django.template.loaders.filesystem.Loader',
+  'django.template.loaders.app_directories.Loader',
 )
 
 MIDDLEWARE_CLASSES = (
